### PR TITLE
Move list plugins function to common place

### PR DESCRIPTION
**Pull request by jieru-hu**
## Motivation
This is duplicate logic, so I want to move them to a common place. 

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes/No

## Test Plan
CI

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: tools/plugin_tests/utils.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```
Comment (omry) [inline tools/plugin_tests/utils.py:1]: tools is meant for complete standalone tools, not for build/nox helpers.
build_helpers it a good place for such logic.

Was there a reason that you merged it without waiting for review?

Comment (jieru-hu) [reply tools/plugin_tests/utils.py:1]: > tools is meant for complete standalone tools, not for build/nox helpers.
> build_helpers it a good place for such logic.
> 
> Was there a reason that you merged it without waiting for review?

I didn't merge this :) 
I created this and then realized that I probably shoudn't do this.

Comment (jieru-hu) [reply tools/plugin_tests/utils.py:1]: thx for the pointer, I will checkout out `build_helpers`

Comment (omry) [reply tools/plugin_tests/utils.py:1]: we could move build helpers to tools, but right now you are starting to spread operational utilities in too many places.

**Pull request closed without merging**